tools/ete/homology_classifier.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="homology_classifier" name="Homology Classifier and Filter" version="@VERSION@.1">
```

Comment:
Can you prepend the file and tool names with `ete_` for consistency? Also the Python one.
And you can remove the `.1` from the version.